### DISCUSSION TITLE
请问一下： AntvDagre 为何不叫Dagre布局呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#6555] [Q&A] [answered]
请问一下：
AntvDagre 为何不叫Dagre布局呢？是否相对于传统的Dagre布局有额外的功能？ 

https://g6.antv.antgroup.com/api/layouts/antv-dagre-layout

**Comment by Rey-Wang** [comment]
可以看下源码，有antv 的特殊逻辑，布局的时候也会考虑 combo

**Comment by Aarebecca** [comment, marked as answer]
AntV Dagre 布局过去是基于 Dagre 布局进行二次开发的，部分实现逻辑上存在差异。
